Documentation/UserInterfaceRecipes.md

```diff
@@ -0,0 +1,136 @@
+# User Interface Recipes
+
+### ViewData (aka ViewModel, Presenter) Objects
```

Comment:
I think we should default to `ViewModel` here since that's been the preferred RAC terminology. (See also: https://github.com/ReactiveCocoa/ReactiveViewModel)